expertsystem/data.py

```diff
@@ -32,12 +33,29 @@ def __eq__(self, other: object) -> bool:
             return self.__value == other.value
         return self.__value == other
 
+    def __gt__(self, other: Any) -> bool:
+        if isinstance(other, Parity):
+            return self.value > other.value
+        return self.value > int(other)
+
+    def __ge__(self, other: Any) -> bool:
+        return self > other or self == other
+
+    def __lt__(self, other: Any) -> bool:
+        return not self >= other
+
+    def __le__(self, other: Any) -> bool:
+        return not self > other
+
```

Comment:
why is this needed?

Reply:
Just to have expected behaviour like:
```python
from expertsystem import io
pdg = io.load_pdg()
{p for p in pdg if p.parity < 0}
```
One could wonder about the `<=` etc but I would have that as well if you introduce this '`int`-like behaviour.

Reply:
I just don't see a use case when someone wants to do such an operation.

Reply:
The `ParticleCollection.filter` function has become a bit of a quantum number search engine (see [interactive exampels](https://pwa.readthedocs.io/projects/expertsystem/en/latest/interactive.html#particle-database)). So I want that function to work as conveniently as possible.
```python
pdg.filter(lambda p: p.parity < 0 and p.spin == 1)
```
But I'm aware that there is a risk: when does convenience become unexpected behaviour?

Reply:
Ok thats maybe a bad example as this would be the same as `p.parity == -1`, but I see your point now.
Like if you want unknown and -1 parity you could do <= 0 or < 1 instead of `parity == -1 and parity == 0` (or None). And its a bit more convenient than having to write `parity.value`

Reply:
This point is however related to #259, which would make `Parity` a more complicated object. Right now, it's just a 'read-only' `int` that can only be ±1.